fastlane/metadata/android/en-US/full_description.txt

```diff
@@ -10,7 +10,8 @@ Supported Platforms:
  - Gaana
  - Youtube
  - Youtube Music
+ - Jio-Saavn
  - (more coming soon)
 
 Note:
- - The availability of YouTube Music in your country is IMPORTANT, if it isn't available consider using a VPN.
\ No newline at end of file
+ - The availability of YouTube Music in your country is IMPORTANT, if it isn't available consider using a VPN.
```

Comment:
also, i havent changed anything in this line. still it's shown. have u faced similar thing before?